### DISCUSSION TITLE
Recovery Codes for TOTP 2FA

### DISCUSSION THREAD

**Discussion by rejochandran** [zitadel/zitadel#8226] [Ideas]
When a user signs up with TOTP as 2FA, it would be great to provide a list of recovery codes that the user can save and can be used to recover 2FA later. 

Is there an alternate way to achieve this or would this be a new feature request?

**Comment by hifabienne** [comment]
Hi @rejochandran 
At the moment we do not have an alternate way to achieve that.
There is already a feature request for that, I recommend you to upvote that issue: https://github.com/zitadel/zitadel/issues/6898

**Reply by rejochandran** [reply]
Thanks for the pointer. Will close this. 🙂
